great_expectations/render/renderer_configuration.py

```diff
@@ -95,49 +97,81 @@ class Config:
         validate_assignment = True
         arbitrary_types_allowed = True
 
-    def __init__(self, **kwargs) -> None:
-        kwargs = RendererConfiguration._set_expectation_type_and_kwargs(kwargs=kwargs)
-        kwargs = RendererConfiguration._set_include_column_name_and_styling(
-            kwargs=kwargs
-        )
-        kwargs["params"] = _RendererParamsBase()
-        super().__init__(**kwargs)
-
-    @staticmethod
-    def _set_expectation_type_and_kwargs(kwargs: dict) -> dict:
-        if "configuration" in kwargs and kwargs["configuration"]:
-            kwargs["expectation_type"] = kwargs["configuration"].expectation_type
-            kwargs["kwargs"] = kwargs["configuration"].kwargs
-        elif (
-            "result" in kwargs
-            and kwargs["result"]
-            and kwargs["result"].expectation_config
+    @root_validator(pre=True)
```

Comment:
Does this need to be a `pre` validator?

Reply:
Actually this is the only one that needed it! I removed it from the rest.